mechanical_turk/questions.xml

```diff
@@ -0,0 +1,38 @@
+<HTMLQuestion xmlns="http://mechanicalturk.amazonaws.com/AWSMechanicalTurkDataSchemas/2011-11-11/HTMLQuestion.xsd">
```

Comment:
@martinbolger , @bshpringer what does this file do? it talks about seahawks game